backends/fs/client.go

```diff
@@ -0,0 +1,63 @@
+package fs
+
+import (
+	"os"
+	"io/ioutil"
+	"path"
+	"path/filepath"
+	"strings"
+)
+
+type Client struct {
+	rootPath    string
+	maxFileSize int64
+}
+
+func NewFsClient() (*Client, error) {
+	// TODO: how to provide rootPath in order to pass tests?
```

Comment:
yeah, but os.Getenv(...) probably isn't the way to go. Maybe custom params for concrete backends in command line arguments? I put there just as a reminder, it's already being used in the `integration/test.sh` to get current testing folder.